### PR TITLE
EB-1405-unit-tests-storeRouteAsync

### PR DESCRIPTION
**Pull request by vdanylov**
Has error response, but payload is correct.

### CODE REVIEW

File: yarn-error.log [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Arguments:
```
Comment (maartenhemmes) [inline yarn-error.log]: @vdanylov please resolve merge conflict

File: src/data/customer/__test__/route.test.js [hunk 2]
```diff
@@ -95,3 +96,52 @@ describe('Retrieve route setting', () => {
     expect('settingRouteSettings' instanceof Array).toBe(true);
   });
 });
+
+describe('Test storeRouteAsync function', () => {
+  const payload = {
```
Comment (maartenhemmes) [inline src/data/customer/__test__/route.test.js]: @vdanylov the unit tests that you have added are still failing:

 error { statusCode: 400,
        statusText: 'Bad Request',
        errorMessage: 
         [ { key: '0',
             messages: 'The routes.0.pickup_date field is required.' },
           { key: '1',
             messages: ' The routes.0.route_locations field is required.' } ] }

**Comment by vdanylov** [comment]
@maartenhemmes where I can check what the problem is?

**Comment by vdanylov** [comment]
@maartenhemmes @vinay3206  I'm not sure where the problem is? Cause all tests which I've added are passed.

File: src/data/customer/__test__/route.test.js [hunk 3]
```diff
@@ -4,6 +4,7 @@ import {
   createRouteLocationAsync,
   storeRouteLocationAsync,
   removeRouteLocationsAsync,
+  storeRouteAsync,
```
Comment (maartenhemmes) [inline src/data/customer/__test__/route.test.js]: I think this function doesn't exist in route.js hence it's failing.

**Pull request closed without merging**